.github/workflows/cypress.yml

```diff
@@ -0,0 +1,42 @@
+name: Run Cypress Tests
+
+on:
+  push:
+    branches:
+      - main 
+  pull_request: 
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checkout the code
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    # Set up Node.js
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: 18 
+
+    # Install dependencies
+    - name: Install dependencies
+      run: npm install
+
+    # Install Cypress
+    - name: Install Cypress
+      run: npm install cypress --save-dev
+
+    # Run Cypress tests
+    - name: Run Cypress Tests
+      run: npx cypress run
+
+    # Save Cypress videos and screenshots for debugging (optional)
+    - name: Upload test artifacts
+      if: failure()
+      uses: actions/upload-artifact@v3
+      with:
+        name: cypress-artifacts
+        path: cypress/videos
```

Comment:
Thank you! The issue likely occurred because the Cypress configuration didn’t enable video recording or the videos directory path was incorrect. I’ll review the Cypress config file and ensure video recording is enabled, or adjust the artifact path to match the actual output directory.